back/app/models/concerns/participation_context.rb

```diff
@@ -18,6 +18,7 @@ module ParticipationContext
   POSTING_METHODS       = %w[unlimited limited].freeze
   VOTING_METHODS        = %w[unlimited limited].freeze
   IDEAS_ORDERS          = %w[trending random popular -new new].freeze
+  IDEAS_ORDERS_EXCLUDE  = %w[trending popular].freeze
```

Comment:
I wonder if we shouldn't add "BUDGETING" or something like that in the name of the constant, so it's clear that these are only excluded for budgeting methods only. We could also consider moving this logic to the participation methods (those from the lib folder).